src/server/api/routes/orders.router.js

```diff
@@ -0,0 +1,66 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const ordersController = require('../controllers/orders.controller');
+
+/**
+ * @swagger
+ * /orders:
+ *  get:
+ *    tags:
+ *    - Orders
+ *    summary: Get all orders of specific user
+ *    description:
+ *      Will return all orders of specific user.
+ *    produces: application/json
+ *    responses:
+ *      200:
+ *        description: Successful request
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.get('/', (req, res, next) => {
+  ordersController
+    .getOrders()
+    .then((result) => res.json(result))
+    .catch(next);
+});
+
+/**
+ * @swagger
+ * /orders/{ID}:
+ *  get:
+ *    tags:
+ *    - Orders
+ *    summary: Get order by user_ID
+ *    description:
+ *      Will return all orders with a matching user_ID.
+ *    produces: application/json
+ *    parameters:
+ *     - in: path
+ *       name: ID
+ *       schema:
+ *         type: integer
+ *         required: true
+ *         description: The ID of the order to get
+ *
+ *    responses:
+ *      200:
+ *        description: Successful request
+ *      5XX:
+ *        description: Unexpected error.
+ *      400:
+ *        description: Bad request. Order ID must be an integer and larger than 0.
+ *      404:
+ *        description: A order with the specified ID was not found.
+ */
+router.get('/:id', (req, res, next) => {
+  ordersController
```

Comment:
we taked about chnging path also this ppass is reserved for takin order by id not by user_id